app/models/concerns/waste_carriers_engine/can_change_registration_status.rb

```diff
@@ -75,6 +75,7 @@ def log_status_change
 
     # Guards
     def renewal_allowed?
+      return false unless %w[ACTIVE EXPIRED].include?(status)
```

Comment:
This line shouldn't be necessary as it's already covered by line 45 – can we try removing it and seeing if it still works?